src/I18n/DateFormatTrait.php

```diff
@@ -278,19 +294,20 @@ public static function parseDateTime($time, $format = null)
                 is_subclass_of(static::class, MutableDate::class);
         }
 
-        $defaultTimezone = static::$_isDateInstance ? 'UTC' : date_default_timezone_get();
+        $timezone = $timezone ?: static::$defaultOutputTimezone;
```

Comment:
This isn't an 'output' but an input/parsing. I thought the new option was for setting the timezone used in string casts/i18nFormat.